### DISCUSSION TITLE
TriggerAnimation doesn't work when using a Path attribute instead of the filename attribute

### DISCUSSION THREAD

**Discussion by evilfactory** [FakeFishGames/Barotrauma#13815] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

The status effect TriggerAnimation doesn't work when using the path attribute instead of a filename attribute, eg:
```xml
<StatusEffect target="Character">
         <TriggerAnimation Type="Walk" filename="funnie" path="%ModDir%/Characters/Human/Animations/ProneWalk.xml" priority="999" ExpectedSpecies="Human" />
</StatusEffect>
```

Debugging the game shows that the else case where the animation is added to the list is never reached when using a ContentPath.

![image](https://github.com/FakeFishGames/Barotrauma/assets/36804725/6c438180-de3e-4787-8928-622b65917cf1)

### Reproduction steps

1. Create a new human animation
2. Inside a status effect put a TriggerAnimation that points to your animation xml
3. Doesn't do anything

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I created a ticket about this in the issue tracker: https://github.com/FakeFishGames/Barotrauma/issues/13816

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, we're going to hotfix it as soon as we can!
